samples/groceryship/src/cancel-request/cancel-request.html

```diff
@@ -2,6 +2,7 @@
   <dv.if condition=!$request.claimed >
     <dv.tx>
       <authentication.authenticate id=$requesterId hidden=true />
+      <dv.link hidden=true />
```

Comment:
For readability, we tend to put the `<dv.link>` last in the tx. Could you move it so that `delete-object` is above `dv.link`?